examples/sqlx_mysql/src/main.rs

```diff
@@ -87,11 +86,9 @@ async fn main() {
                 .into(),
             date!(2020 - 8 - 20).with_time(time!(0:0:0)).into(),
         ])
-        .build(MysqlQueryBuilder);
+        .build_sqlx(MysqlQueryBuilder);
```

Comment:
Quick question. Where is this `build_sqlx` defined? Why I don't seem to find it?

Reply:
In the binder crate